clients/go/cmd/zbctl/internal/commands/root.go

```diff
@@ -201,8 +202,9 @@ func keyArg(key *int64) cobra.PositionalArgs {
 	}
 }
 
-func printJSON(value interface{}) error {
-	valueJSON, err := json.MarshalIndent(value, "", "  ")
+func printJSON(value proto.Message) error {
```

Comment:
This method is duplicated in `status.go`, can you remove it please (the status one)?

Reply:
Done